.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`.idea` is folder which contains your personal IDE configs. Looks like we should hide this folder as `out` or `target` folders.

Reply:
Okay, will do it.